hasura/metadata/databases/default/tables/public_circles.yaml

```diff
@@ -120,6 +120,12 @@ select_permissions:
             - read_circle:
                 _eq: true
       limit: 5
+  - role: discord-bot
+    permission:
+      columns:
+        - id
+        - name
```

Comment:
Do we need to add here permission to the `users` table or is that covered with [this](https://github.com/coordinape/coordinape/pull/1740/files#diff-81f3eaa81775090456210ed3aef418934147e91a8299297a24e0e9675cf476f9R116)?

Also wondering about access to the events for [these notifications](https://github.com/coordinape/discord-bot/issues/28) such as nominations, vouches, user opts out, etc.

Reply:
@karelianpie it is covered.

You can't add relationship permissions, just column permissions. A relationship is allowed to be utilized by a hasura role when both tables in a relationship are visible to the user.

> Also wondering about access to the events for https://github.com/coordinape/discord-bot/issues/28 such as nominations, vouches, user opts out, etc.

We shouldn't need perms for those, since cron tasks have access to the admin-level client. and will have access to the entirety of the public and discord schemas.